koans/about_dice_project.rb

```diff
@@ -3,7 +3,19 @@
 # Implement a DiceSet Class here:
 #
 # class DiceSet
-#   code ...
+class DiceSet
+
+  attr_reader :values
+
+  def roll(numDie)
+
+    @values = []
+
+    numDie.downto(1) { |i|
```

Comment:
Thanks, I will change it to numDie.downto(1) { |i|  @values.push(1 + rand(6)) }  
because:
Curly braces usually enclose a single-line code, with the method and its block written on one line. 
If the block code is too long or large, instead of the parentheses on the line with the method call,

"do" is left, and the code is written from a new line indented and at the end is "closed" by the "end" .